### PR TITLE
TinyMCE 5.8 Docs release PR

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-758

Description of Changes:
* TinyMCE 5.8 Docs release PR

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/misc/supported_versions.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Supported versions of {{site.productname}}:
 
 | Version |   Release Date    |  End of Support   |
 |:-------:|:-----------------:|:-----------------:|
-|   5.7   | February 24, 2021 |        TBD        |
+|   5.8   | TBD               |        TBD        |
```
Comment (lnewson) [inline _includes/misc/supported_versions.md:7]: Just adding a reminder we need to add a date here before merging

- [ ] Add release date

File: release-notes/release-notes58.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+- [Known issues](#knownissues)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## New features
+
+The following new features were added for the {{site.productname}} 5.8 release.
+
+### Improved keyboard navigation for the inline boundary feature
```
Comment (lnewson) [inline release-notes/release-notes58.md:29]: This was placed in the wrong place and we missed it in the review. The should be in the enhancements section instead.

**Review by ashfordneil — APPROVED** [review]

File: advanced/configuring-comments-callbacks.md [hunk 3]
```diff
@@ -126,13 +126,36 @@ The done callback needs to take an object of the form:
...
+The `tinycomments_resolve` function is passed a (`req`) object as the first parameter, which contains the following key-value pair:
+
+* **conversationUid**: The uid of the conversation the reply is targeting.
+
+The `done` callback should accept the following object:
+
+```js
+{
+  canResolve: boolean // whether or not the conversation can be resolved
+  reason: string? // an optional string explaining why resolving was not allowed (if canResolve is false)
```
Comment (JamesToohey) [inline advanced/configuring-comments-callbacks.md:146]: Right?

```suggestion
  reason?: string // an optional string explaining why resolving was not allowed (if canResolve is false)
```

**Review by JamesToohey — APPROVED** [review]

File: release-notes/release-notes58.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes58.md]: Fixed here: [`c25796c` (#1966)](https://github.com/tinymce/tinymce-docs/pull/1966/commits/c25796c80db5ed5998f66c8d759a510435df5475)

**Review by lnewson — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**